.github/ISSUE_TEMPLATE

```diff
@@ -4,25 +4,14 @@ Provide a description of what is the current problem and why you are raising thi
 If it's a bug please describe what was the unexpected thing that occured and what was the
 expected behaviour.
 
-## Solution
-
-Here provide a detailed explanation, if you can suggest one, for how to go about solving the problem.
-
-
-## Tasklist
-
-Here provide a checkpoint list of tasks that need to be done in order for this issue to be considered closed.
-This will help us evaluate if the issue has been properly addressed.
-
-Example:
-
-- [ ] Implement feature XXX
-- [ ] Create test for XX
-- [ ] Add a changelog entry.
-
-## Only For Bugs
+### System Description
 
-Bother with this section only if you are reporting a bug
+Here add a detailed description of your system, e.g. output of the following script:
 
-### System Description
-Here add a detailed description of your system, e.g. output of `raiden version`, `geth version`, `solc --version` e.t.c.
+```
+uname -a
+command -v solc && solc --version
+command -v geth && geth version
+command -v parity && partiy --version
```

Comment:
```diff
-command -v parity && partiy --version
+command -v parity && parity --version
```

Reply:
?

Reply:
You have a typo because you like to party ;) . Look at the diff.

Reply:
AHHh, I totally missed it, thanks.